src/App.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Board from './components/Board';
+
+// Any alive cell that is touching less than two alive neighbours dies.
+// Any alive cell touching four or more alive neighbours dies.
+// Any alive cell touching two or three alive neighbours does nothing.
+// Any dead cell touching exactly three alive neighbours becomes alive.
+
+//number of rows/columns
+const size = 40;
```

Comment:
try to use a more descriptive variable name, this variables should be named SIZE_ROWS_COLUMNS